test/cql-pytest/test_keyspace.py

```diff
@@ -97,15 +97,19 @@ def test_drop_keyspace_nonexistent(cql):
         cql.execute('DROP KEYSPACE nonexistent_keyspace')
 
 # Test trying to ALTER a keyspace.
-def test_alter_keyspace(cql, this_dc):
+def test_alter_keyspace(cql, this_dc, has_tablets):
     with new_test_keyspace(cql, "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 1 }") as keyspace:
-        cql.execute(f"ALTER KEYSPACE {keyspace} WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 3 }} AND DURABLE_WRITES = false")
+        repl_opts = "" if has_tablets else f" REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 3 }} AND "
+        cql.execute(f"ALTER KEYSPACE {keyspace} WITH {repl_opts} DURABLE_WRITES = false")
 
 # Test trying to ALTER a keyspace with invalid options.
-def test_alter_keyspace_invalid(cql, this_dc):
+def test_alter_keyspace_invalid(cql, this_dc, has_tablets):
     with new_test_keyspace(cql, "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 1 }") as keyspace:
         with pytest.raises(ConfigurationException):
-            cql.execute(f"ALTER KEYSPACE {keyspace} WITH REPLICATION = {{ 'class' : 'NoSuchStrategy' }}")
+            if has_tablets:
+                cql.execute(f"ALTER KEYSPACE {keyspace} WITH tablets = {{'initial': 'yes please'}}")
+            else:
+                cql.execute(f"ALTER KEYSPACE {keyspace} WITH REPLICATION = {{ 'class' : 'NoSuchStrategy' }}")
```

Comment:
Again, I would avoid changing this failing test to pass (I don't want to repeat the reasoning I already explained above).

Reply:
Reverted the changes and marked with xfail instead.